README.rst

```diff
@@ -0,0 +1,16 @@
+BAP IDA Python
+==============
+
+This package provides the necessary IDAPython scripts required for
+interoperatibility between BAP and IDA Pro.
+
+Installation
+------------
+
+All of the requisite scripts can be installed via ``pip`` using::
+
+    pip install git+git://github.com/BinaryAnalysisPlatform/bap-ida-python
+
+Now all that remains is to inform IDA about where the scripts are and how to access BAP. This process is simplified greatly using the plugin installer. The installer can be run using::
+
+    python -c 'from bap_ida_python.loader.installer import main; main()'
```

Comment:
It is packed as a python script, but since it is run only once, I did not think it made sense to have it install a new command itself onto the shell PATH.

Also, I initially packed it in as a post-install command. However, usually the `pip` command is run via `sudo` and in such a scenario, `bap` is not really on the `$PATH` if installed through `opam`.

This script is interactive only of the person does not have the `which` command on their system. Otherwise, it automatically generates all necessary information.